framework/src/main/java/org/apache/drill/test/framework/TestDriver.java

```diff
@@ -563,6 +633,16 @@ else if(timeoutFailures.contains(test)){
     return totalExecutionFailures + totalDataVerificationFailures + totalPlanVerificationFailures + totalTimeoutFailures + totalRandomFailures;
   }
 
+  private static boolean matches(String actual, String expected) {
```

Comment:
Can you check if there is an existing matches functions in Utils? If not move this there?